tests/mock_clients/mock_s3.py

```diff
@@ -66,6 +67,16 @@ def get(self):
         else:
             return {"key": self.path}
 
+    def load(self):
+        if not self.path.exists() or self.path.is_dir():
+            raise ClientError({}, {})
+        else:
+            return {"key": str(self.path)}
+
+    @property
+    def key(self):
+        return str(self.path.relative_to(self.root))
```

Comment:
Tests might be failing because you're calling `str` here. I suspect it's a `PosixPath` vs `WindowsPath` thing.

Reply:
Fixed 👍